### PR TITLE
Enable Dependabot for the repository

### PR DESCRIPTION
**Pull request by jglick**
I requested access for Dependabot to this repo, but I am not an admin of it (only have write access); any way to find out who _is_ an admin?

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
> any way to find out who is an admin?

You should be able to see members or @jenkinsci/github-admins . It was created few week ago after the request by @casz

File: pom.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
         <module>global-configuration</module>
         <module>scripted-pipeline</module>
         <module>global-shared-library</module>
+        <!-- if adding new archetypes please also update .dependabot/config.yml -->
```
Comment (jglick) [inline pom.xml]: pending https://github.com/dependabot/feedback/issues/500

**Comment by oleg-nenashev** [comment]
Enabled Dependabot for the repo

**Comment by jglick** [comment]
And here they come, OMG. I thought there was some sort of default limit on how many new PRs get filed for a given repo. :-)

**Comment by oleg-nenashev** [comment]
Enjoy :P

**Pull request merged**